### ISSUE TITLE
Can you change the link text of a param that uses a namepath?

### ISSUE THREAD

**Issue by Native-Coder** [jsdoc3/jsdoc#1352]
Is it possible to change the link text in a @param tag like you do with the @link tag? I tried the @link syntax but that didn't work. Here's the use case...

I have a method that accepts a typedef parameter. That typedef is in the same module as the class that the method belongs to. So to document the parameter, I do this
`@param {module:InventoryManager~PartData} Data - The data to be added or updated for the transaction part with the id = Data.id`

Which looks horrible in the documentation. I would like to show just a "PartData" link in the docs, instead of a link that says "module:InventoryManager~PartData"

I tried
`@param [PartData]{module:InventoryManager~PartData} Data - The data to be added or updated for the transaction part with the id = Data.id`

to no avail.

**TLDR;**

I would like to do something like this
`[PartData]{@link module:InventoryManager~PartData}`
With a @param tag. Is that possible?

**Comment by hegemonic** [comment]
Sorry, but this isn't currently possible. When #738 is fixed, it will be possible to tell JSDoc to only show the symbol's name (`PartData`) rather than the full longname (`module:InventoryManager~PartData`). I don't have an ETA for fixing that issue.
